documentation/Database-Schema-Review.md

```diff
@@ -0,0 +1,99 @@
+
+## Tables
+
+### review
+
+A single review is created per pair of **application-stage**. Although some applications may not required any review (which will be defined by the actions for the particular template and stage on the trigger `onApplicationSubmitted`).
+
+The `application_id` links to the application being reviewed.
+
+The `status` of the review is either: `'Awaiting review'`, `'In Progress'`, `'Ready'`, `'Approvable'`, `'Non-Approvable'`. This is the status of the whole review. It is automatically defined by the combination of `review_section.decision` - the review of each section. See [SQL functions](Functions.md) for more details.
+
+The `comment` is some overall comment for the application - added by the Reviewer making the consolidation. It will be sent to the Applicant with the review.
+
+The `time_created` is the time when the review has started. The created is when the first section is assigned to a reviewer.
+
+The `trigger` is updated everytime the review has changes done by users or a scheduler. See more about [triggers](Triggers-and-Actions.md).
+
+**To be considered:**
+
+- The `review` is per pair of **application-stage** or one per each time it goes back to the Applicant? I think it would be always one per each pair, although the field `review.comment` implies one overall comment which should be sent to the Applicant - per review. In the case if the review is re-submitted and another review will be started (on the same stage), how we would we keep this comment? Or do we remove it or do we create another `review`?
+- The `status` of `'Ready'` - which is when a lower level reviewer has finished the review and the consolidation can happen - is defined in the Schema latest documentation of Drive as `'Reviewed'`. Is it what we have agreed on? I can update it, just want to check that first.
+
+### review section join
+
+The `review_id` links to the main review record of the **application-stage** pair.
+
+The `review_section_assignment_id` links to the reviewer of the section.
+
+The `review_section_id` links to the section being reviewed.
+
+The `send_to_applicant` is a flag for adding this section (and `review_section.comment`) to be sent back to the Applicant.
+
+**To be considered:**
+
+- Can we change `review_section_assignment` to `review_assignment`. Just to keep it short. We know that there will be 1 section per 1 review assignment, the table don't need to have such a long name. And when refereced in `review_section_join`
+  the `section` is duplicated in `review_section_join.review_section_assignment_id`.
+
+- The field `section` linking to the `Application_section` in both `review_section_assignment` and `review_sectiob_join`. One has to go.
```

Comment:
I don't quite understand -- it looks like `review_section_join` links to the *review* section, and the `review_section_assignment` links to the *application* section. Is that right?